src/chainstate/coordinator/mod.rs

```diff
@@ -589,6 +598,43 @@ impl<'a, T: BlockEventDispatcher, N: CoordinatorNotices, U: RewardSetProvider>
                     increment_stx_blocks_processed_counter();
                     let block_hash = block_receipt.header.anchored_header.block_hash();
 
+                    let mut attachments_instances = HashSet::new();
+                    let atlas_config = AtlasConfig::default();
```

Comment:
Shouldn't an atlas config instance be a member of the chains coordinator?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/4bb53af65793fcd49fb2cc23b6dfe14deb725eeb